### PR TITLE
chore(doc): add status to README in prep for archiving

### PR DESCRIPTION
**Pull request by rvagg**
Will archive after merging

### CODE REVIEW

**Comment by oed** [comment]
Could you change the note to say that this package is now part of the Ceramic code base? 
https://github.com/ceramicnetwork/js-ceramic

cc @ukstv

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # IPFS Pinning Aggregation
 
+## Status
+
+This repository is in a **frozen** state. It is not being maintained or kept in sync with the tools and libraries it builds on. Even though work on this repository has been **shelved**, anyone interested in updating or maintaining this project should express their interest on one Filecoin community conversation mediums: <https://github.com/filecoin-project/community#join-the-community>.
```
Comment (rvagg) [inline README.md]:
```suggestion
This repository is in a **frozen** state. It is not being maintained or kept in sync with the tools and libraries it builds on. This package has now been largely integrated into <https://github.com/ceramicnetwork/js-ceramic> and is being maintained there. Even though work on this repository has been **shelved**, anyone interested in updating or maintaining this as a stand-alone project should express their interest on one Filecoin community conversation mediums: <https://github.com/filecoin-project/community#join-the-community>.
```

**Comment by rvagg** [comment]
Let me know what you think of the suggested change, or feel free to suggest something else - or even another approach to dealing with this repo! The main aim here is in avoiding stale code and giving potential users a misconception about the state of the code here.

File: README.md (hunk 1 above)
Comment (oed) [reply README.md]: lgtm :)

**Comment by oed** [comment]
That's good to me @rvagg. Thanks for updating!

**Pull request merged**